ios-drag-drop.js

```diff
@@ -1,12 +1,15 @@
 (function(doc) {
-function _exposeIosHtml5DragDropShim(config) {
 
+  
+function _exposeIosHtml5DragDropShim(config) {
+  var shimConfig;
   log = noop; // noOp, remove this line to enable debugging
 
   var coordinateSystemForElementFromPoint;
 
   function main() {
     config = config || {};
+    shimConfig = config;
```

Comment:
You are right...this config object wasn't available to all places of the shim before.  I'll update this now.